jobrunner/sync.py

```diff
@@ -19,13 +19,14 @@
 
 
 def main():
-    log.info(
-        f"Polling for JobRequests at: "
-        f"{config.JOB_SERVER_ENDPOINT.rstrip('/')}/job-requests/"
-    )
-    while True:
-        sync()
-        time.sleep(config.POLL_INTERVAL)
+    with set_log_context(prefix='sync'):
```

Comment:
:+1: This looks like a reasonable re-use of the existing logging stuff to get what we want for now. I wonder if it would make sense to do this inside `service.py` though to localise the hack, as it were. It would mean wrapping `sync.main` in another function to use as the thread target, but would only be a couple of lines.

Reply:
Ok, that makes sense for now